src/main/headless/validation.ts

```diff
@@ -4,7 +4,7 @@ import { StandaloneSubcommand } from "./subcommand";
 export class Validation extends StandaloneSubcommand {
   public isValidPrivateKey(privateKeyHex: string): boolean {
     try {
-      this.execSync("validation", "private-key", privateKeyHex);
+      this.execSync(["validation", "private-key", privateKeyHex]);
```

Comment:
Is it intended?

```suggestion
      this.execSync(["validation", "private-key"], privateKeyHex);
```

Reply:
Yes, it was intended but the code you suggested is correct and right. But `NineChronicles.Headless.Executable validation private-key` doesn't support stdin check. It should be implemented in **planetairum/NineChronicles.Headless** first.